core/src/main/scala/cats/data/Const.scala

```diff
@@ -39,6 +39,12 @@ final case class Const[A, B](getConst: A) {
 object Const extends ConstInstances {
   def empty[A, B](implicit A: Monoid[A]): Const[A, B] =
     Const(A.empty)
+
+  class partialApply[B] {
```

Comment:
Perhaps change this to something like `OfPartiallyApplied` to be consistent with https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/EitherT.scala#L275